code/modules/antagonists/vampire/vampire_powers/dantalion_powers.dm

```diff
@@ -0,0 +1,198 @@
+/proc/isvampirethrall(mob/living/M)
+	return istype(M) && M.mind && M.mind.has_antag_datum(/datum/antagonist/mindslave/thrall)
+
+/datum/vampire_passive/increment_thrall_cap/on_apply(datum/antagonist/vampire/V)
+	V.subclass.thrall_cap += 1
+	gain_desc = "You can now thrall one more person. Up to a maximum of [V.subclass.thrall_cap]"
+
+/datum/vampire_passive/increment_thrall_cap/two
+
+/datum/vampire_passive/increment_thrall_cap/three
+
+/obj/effect/proc_holder/spell/vampire/enthrall
+	name = "Enthrall (150)"
+	desc = "You use a large portion of your power to sway those loyal to none to be loyal to you only."
+	gain_desc = "You have gained the ability to thrall people to your will."
+	action_icon_state = "vampire_enthrall"
+	required_blood = 150
+	deduct_blood_on_cast = FALSE
+
+/obj/effect/proc_holder/spell/vampire/enthrall/create_new_targeting()
+	var/datum/spell_targeting/click/T = new
+	T.range = 1
+	T.click_radius = 0
+	return T
+
+/obj/effect/proc_holder/spell/vampire/enthrall/cast(list/targets, mob/user = usr)
+	var/datum/antagonist/vampire/vampire = user.mind.has_antag_datum(/datum/antagonist/vampire)
+	for(var/mob/living/target in targets)
+		user.visible_message("<span class='warning'>[user] bites [target]'s neck!</span>", "<span class='warning'>You bite [target]'s neck and begin the flow of power.</span>")
+		to_chat(target, "<span class='warning'>You feel the tendrils of evil invade your mind.</span>")
+		if(do_mob(user, target, 5 SECONDS))
+			if(can_enthrall(user, target))
```

Comment:
I just noticed this is mostly copy-pasted from the other file. Previous behaviour was that if the vamp couldn't enthral then he got his blood back. Currently, this ain't the case. Prob would want to add an else for this if you want to keep that functionality.

Reply:
this still is the case, due to the fact that the spell has `deduct_blood_on_cast = FALSE`

Reply:
Ah yes of course. I missed that part!